frontend/components/notifications/management/NotificationsManagementSubscriptionDialog.vue

```diff
@@ -24,31 +17,32 @@ function closeDialog(): void {
 const checkboxes = ref({
   is_attestations_missed_subscribed: props.value?.is_attestations_missed_subscribed ?? false,
   is_block_proposal_subscribed: props.value?.is_block_proposal_subscribed ?? false,
-  is_group_offline_subscribed: props.value?.is_group_offline_subscribed ?? false,
   is_max_collateral_subscribed: props.value?.is_max_collateral_subscribed ?? false,
   is_min_collateral_subscribed: props.value?.is_min_collateral_subscribed ?? false,
-  is_real_time_mode_enabled: props.value?.is_real_time_mode_enabled ?? false,
   is_slashed_subscribed: props.value?.is_slashed_subscribed ?? false,
   is_sync_subscribed: props.value?.is_sync_subscribed ?? false,
   is_upcoming_block_proposal_subscribed: props.value?.is_upcoming_block_proposal_subscribed ?? false,
   is_validator_offline_subscribed: props.value?.is_validator_offline_subscribed ?? false,
   is_withdrawal_processed_subscribed: props.value?.is_withdrawal_processed_subscribed ?? false,
 })
 const thresholds = ref({
-  group_offline_threshold: formatFraction(props.value?.group_offline_threshold ?? 0),
   max_collateral_threshold: formatFraction(props.value?.max_collateral_threshold ?? 0),
   min_collateral_threshold: formatFraction(props.value?.min_collateral_threshold ?? 0),
 })
 const emit = defineEmits<{
-  (e: 'change-settings', settings: Omit<NotificationSettingsValidatorDashboard, 'is_webhook_discord_enabled' | 'webhook_url'>): void,
+  (e: 'change-settings', settings: Omit<NotificationSettingsValidatorDashboard,
+  'group_offline_threshold'
+  | 'is_group_offline_subscribed'
+  | 'is_real_time_mode_enabled'
+  | 'is_webhook_discord_enabled'
+  | 'webhook_url'>): void,
```

Comment:
in the future it would be nice to work together in such `corner cases` or it would be good if the `frontend team` could also `remove` the corresponding `backend code` 👉 `noob alert` (we would need some onboarding 🙈